beets/autotag/mb.py

```diff
@@ -228,7 +232,7 @@ def album_info(release):
                 index,
                 int(medium['position']),
                 int(track['position']),
-                len(medium['track-list']),
+                total,
```

Comment:
… use `len(all_tracks)` here instead. `all_tracks` is a good variable name (it contains all the tracks), and `len()` is a known function (or should be, anyway).